pkg/columns/templates.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 The Inspektor Gadget authors
```

Comment:
I think I would call the the second one `pid` instead of just `id` - and I think `uid` (and other non-pid ids) should be handled differently (they're usually not as high as pids get, right?).